projects/1.18/assets/butchercraft/lang/zh_cn.json

```
@@ -0,0 +1,201 @@
+  "block.butchercraft.jelly_orange_block": "橘子果冻",
+  "block.butchercraft.jelly_pink_block": "草莓果冻",
+  "block.butchercraft.jelly_purple_block": "葡萄果冻",
+  "block.butchercraft.jelly_red_block": "樱桃果冻",
+  "block.butchercraft.jelly_white_block": "椰子果冻",
+  "block.butchercraft.jelly_yellow_block": "柠檬果冻",
+  "block.butchercraft.lamb_block": "生羊肉块",
+  "block.butchercraft.mashed_potato_gravy_block": "肉汁土豆泥",
+  "block.butchercraft.meat_hook": "挂肉钩",
+  "block.butchercraft.meat_pie": "肉馅饼",
+  "block.butchercraft.pig_hide_carpet": "猪皮",
+  "block.butchercraft.pork_block": "生猪肉块",
+  "block.butchercraft.potroast_block": "罐焖牛肉",
+  "block.butchercraft.pulled_pork_block": "手撕猪肉",
+  "block.butchercraft.rack_lamb_block": "烤羊排",
+  "block.butchercraft.salisbury_steak_block": "索尔兹伯里牛排",
+  "block.butchercraft.sheep_hide_carpet": "羊皮",
+  "block.butchercraft.stirfry_block": "蔬菜炒肉丝",
+  "butchercraft.advancement.bonesaw.desc": "制作一把骨锯",
+  "butchercraft.advancement.bonesaw.name": "骷髅的噩梦。",
+  "butchercraft.advancement.butcherknife.desc": "制作一把屠刀",
+  "butchercraft.advancement.butcherknife.name": "屠夫不杀五爪猪",
+  "butchercraft.advancement.cannibalism.desc": "试试去宰了一个人类。",
+  "butchercraft.advancement.cannibalism.name": "这里不是《环世界》……",
+  "butchercraft.advancement.cow.desc": "对一头牛使用屠刀（右击）",
+  "butchercraft.advancement.cow.name": "我牛肉呢？",
+  "butchercraft.advancement.everything.desc": "品尝所有肉类美食",
+  "butchercraft.advancement.everything.name": "每样都来一点",
+  "butchercraft.advancement.everything_plus.desc": "品尝所有熟透的可食用部分。",
+  "butchercraft.advancement.everything_plus.name": "我全都要！",
+  "butchercraft.advancement.gutknife.desc": "制作一把穿肠刀",
+  "butchercraft.advancement.gutknife.name": "有胆量，小子。",
+  "butchercraft.advancement.heart.desc": "吃一颗生的心脏",
+  "butchercraft.advancement.heart.name": "贪心不足蛇吞象",
+  "butchercraft.advancement.hook.desc": "制作一个挂肉钩",
+  "butchercraft.advancement.hook.name": "如挂钩尖",
+  "butchercraft.advancement.pig.desc": "对一头猪使用屠刀（右击）",
+  "butchercraft.advancement.pig.name": "健康又美味",
+  "butchercraft.advancement.root.desc": "欢迎来到屠宰工艺！",
+  "butchercraft.advancement.root.name": "屠宰工艺",
+  "butchercraft.advancement.sheep.desc": "对一头羊使用屠刀（右击）",
+  "butchercraft.advancement.sheep.name": "香啊，很香啊",
+  "butchercraft.advancement.skinningknife.desc": "千钧一发",
+  "butchercraft.advancement.skinningknife.name": "制作一把剥皮刀",
+  "butchercraft.advancement.whole_cow.desc": "品尝每一种熟透的牛肉",
+  "butchercraft.advancement.whole_cow.name": "除了哞眸叫以外，什么都有",
+  "butchercraft.advancement.whole_pig.desc": "品尝每一种熟透的猪肉",
+  "butchercraft.advancement.whole_pig.name": "除了哼哼叫以外，什么都有",
+  "butchercraft.advancement.whole_sheep.desc": "品尝每一种熟透的羊肉",
+  "butchercraft.advancement.whole_sheep.name": "除了咩咩叫以外，什么都有",
+  "butchercraft.butcherknife.rightclick": "右击以屠宰",
+  "item.butchercraft.bbq_jar_item": "烤肉酱",
+  "item.butchercraft.bbq_ribs": "盘装炭烤排骨",
+  "item.butchercraft.beef_jerky": "牛肉干",
+  "item.butchercraft.beef_rib": "生肋条",
+  "item.butchercraft.beef_ribs": "生牛排骨",
```

```suggestion
  "item.butchercraft.beef_ribs": "生原切牛排骨",
```
为了与上方的条目作区别，加上“原切”两字